lumen/ai/views.py

```diff
@@ -123,14 +122,8 @@ async def _render_component(self):
                 value=True, name="Executing SQL query...", height=50, width=50
             )
 
-        source = memory["current_source"]
-        memory["current_table"] = self.name
-        source.add_table(self.name, self.spec)
+        pipeline = memory["current_pipeline"]
```

Comment:
This is already set in https://github.com/holoviz/lumen/pull/595/files#diff-d56253257f94b517eb742b1213dec9b7ccb3fafa7416be4044987ccd54ed3c83R561